### DISCUSSION TITLE
Unable to track goals using hash routing in Ghost

### DISCUSSION THREAD

**Discussion by OutpostRyan** [plausible/analytics#1839] [Anything Else]
Hi!

I'm trying to set up some goal tracking on a Ghost-powered site, which uses hash based routing for some of its member management. But I don't see any events recorded. I'm using the hosted version of Plausible.

The script is loading using the hash option: "script.hash.outbound-links.js"

I've set up these goals

![image](https://user-images.githubusercontent.com/83042054/163850525-818fa433-0243-4a1d-b018-df15f0936601.png)

but visits to urls like https://www.levernews.com/#/portal/signup don't register. I initially setup the goals with the longer url snippet, but that didn't appear to work either.

Other goals on more traditional routing are registering fine, as are web visits generally.

Is there something I'm doing incorrectly?

Thanks

**Comment by RobertJoonas** [comment]
Hi @OutpostRyan!

I just visited the link myself. Plausible nicely sent a pageview and persisted the hash part of the URL. You can see the `/#/portal/signup` page path showing up in the Top Pages section. To add it as a pageview goal, please use this exact same path. Then you should be able to see the pageview goal in Goal Conversions too. You can read more about pageview goals here: https://plausible.io/docs/pageview-goals

**Comment by OutpostRyan** [comment]
Absolutely, it showed up. I think I might have given up too soon on
the /#/portal/signup/ formatted goals.

Thanks so much for checking and apologies for the user error!

Ryan
